### ISSUE TITLE
TV safe area

### ISSUE THREAD

**Issue by clbr** [Huntereb/Awoo-Installer#155]
Parts of the text are outside the TV safe area, affecting docked mode. For example on my TV, on the internet screen, I can't see which button it says to use for URLs.

**Comment by clbr** [comment]
Perhaps so, mine is a 10 year old LCD. Still, the correct practice is to account for the safe TV area when targeting TVs.

Slightly offtopic:
I ended up not using Awoo for other reasons. Installing big titles crashed instantly with 2168-0001, not using applet mode wasn't possible due to chicken-egg issues (need a title installed to use game mode, but can't install titles), and installing small titles (<500mb) claimed success, but they weren't visible anywhere. Nothing on home screen or settings->software, and the SD card's free space hadn't changed.

All this was via http, on fw 10.2, atmosphere 1.1.1.

I did doubt the SD card at that point, but it's new, freshly formatted fat32 and genuine. So I tried other installers, and dbi installed everything successfully, in applet mode, via http, and even supported the html index page. It's closed source however.
